ng/src/web/pages/groups/dialog.js

```diff
@@ -30,103 +31,117 @@ import Layout from '../../components/layout/layout.js';
 
 import PropTypes from '../../utils/proptypes.js';
 
-import withDialog from '../../components/dialog/withDialog.js';
+import SaveDialog from '../../components/dialog/savedialog.js';
 
 import Checkbox from '../../components/form/checkbox.js';
 import FormGroup from '../../components/form/formgroup.js';
 import MultiSelect from '../../components/form/multiselect.js';
 import TextField from '../../components/form/textfield.js';
 
 const Dialog = ({
-    all_users,
-    comment,
+    allUsers,
     grant_full,
-    id,
-    name,
-    users,
-    onValueChange,
+    group,
+    title = _('New Group'),
+    visible,
+    onClose,
+    onSave,
   }) => {
+  const is_edit = is_defined(group);
+
+  const defaults = {name: _('Unnamed'), users: []};
```

Comment:
Could be moved out of the Dialog component into a constant at the module scope. Same as in the other dialog.